Formula/re-flex.rb

```diff
@@ -0,0 +1,22 @@
+class ReFlex < Formula
+  desc "The regex-centric, fast and flexible scanner generator for C++"
+  homepage "https://www.genivia.com/doc/reflex/html"
+  url "https://github.com/Genivia/RE-flex/archive/reflex-1.0.12.tar.gz"
+  sha256 "75a1761d9a6fd3fee6454e3b6ec954efbad76ece9b68633a07ada977af4cd300"
+
+  depends_on "boost"
+
+  def install
+    ENV["YACC"] = "#{Formula["bison"].bin}/bison -y"
+    # system "autoreconf", "-fi"
+    system "./configure",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "-j", "install"
+  end
+
+  test do
+    system "#{bin}/reflex", "--version"
```

Comment:
Is it possible to create a test that uses the functionality of the software rather than checking the version number ?

Reply:
Its more about testing the compilation that Homebrew does so we wouldn't expect a full suite of tests like for instance the test that are in the gcc formula https://github.com/Homebrew/homebrew-core/blob/master/Formula/gcc.rb#L120-L130 .